.github/workflows/publish.yml

```diff
@@ -0,0 +1,60 @@
+name: 🚀 Publish a Dataset
+run-name: "🚀 Publish a Dataset: ${{ inputs.product }}"
+
+on:
+  workflow_dispatch:
+    inputs:
+      product:
+        description: "Which product to publish"
+        type: choice
+        required: true
+        options:
+          - cbbr
+          - checkbook
+          - colp
+          - cpdb
+          - developments
+          - facilities
+          - pluto
+          - zoningtaxlots
+      draft_build:
+        description: "Build folder in drafts to publish"
+        type: string
+        required: true
+      publishing_version:
+        description: "Version of official release"
+        type: string
+        required: true
+      product_acl:
+        description: "Set access control list for output in DO"
+        type: choice
+        required: true
+        default: public-read
+        options:
+          - public-read 
+          - private
+
+jobs:
+  publish:
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: nycplanning/build-base:latest
+    env:
+      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          sparse-checkout: 'dcpy'
+
+      - name: publish
+        run: |
+          python3 -m dcpy.connectors.edm.publishing publish \
+            db-$${{ inputs.product }} \
+            $${{ inputs.draft_build }} \
```

Comment:
@fvankrieken what effect does doubling the $ have?

Reply:
Oops